CMakeLists.txt

```diff
@@ -15,10 +15,12 @@ if(CMAKE_BUILD_TYPE MATCHES "Release")
 	endif()
 endif()
 
+set(_gnulike GNU Clang)
```

Comment:
Is this variable necessary if it's only used in one place?

Reply:
yes that's how CMake needs to have it if not using a Regex